testbot_description/launch/displayHalfPhantom.launch

```diff
@@ -0,0 +1,14 @@
+<launch>
+<group ns="halfphantom">
+<arg name="model" />
+<arg name="gui" default="False" />
+<arg name="vel" default="True" />
+<arg name="eff" default="True" />
+<param name="robot_description" textfile="$(arg model)"  />
+<param name="use_gui" value="$(arg gui)"/>
+<node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
+<node name="tf_simpleTransform_broadcaster" pkg="testbot_description" type="tf_simpleTransform_broadcaster" />
+<param name="publish_default_velocities"  value="$(arg vel)" />
+<param name="publish_default_effort"  value="$(arg eff)" />
+</group>
+</launch>
```

Comment:
Please indent all the `.xml` files in a sane way, so it is easier in the future to read them and modify them (i.e. maintainability) . Choose the style that you prefer (ROS documentation seems to indent new tags with a 2-space indentation, see http://wiki.ros.org/roslaunch/XML#Example_.launch_XML_Config_Files) but be consistent.